docs/accessibility.md

```diff
@@ -0,0 +1,221 @@
+# Accessibility
+
+## General Practices
+
+As a general rule, accessible software is achieved by following web development best practices. As Treeherder uses the `reactstrap` library, it is recommended using one of its components, if possible.
+
+If you can not find a suitable component, prefer [HTML5 semantic tags](https://developer.mozilla.org/en-US/docs/Glossary/Semantics#Semantics_in_HTML) and avoid using generic tags, such as `span` and `div`. Use those only when there is no other more semantic tag. Moreover, try to use more **headings** (`h1` to `h6`) whenever possible, because screen readers users usually navigate by headings and that is only possible when developers insert them in the code.
+
+If you can build your layout neither with `reactstrap` nor with semantic tags, please insert an `aria-role`. Down below, there are some elements constructed that way in Treeherder:
+
```

Comment:
This wording is a little unclear to me. Since you mentioned that there are elements constructed this way in Treeherder, I was expecting to click on a link to show some Treeherder code to show how it was done. You don't need to do that but can you reword this?